components/nimbus/src/enrollment.rs

```diff
@@ -134,10 +134,26 @@ impl Display for DisqualifiedReason {
     }
 }
 
-// Every experiment has an ExperimentEnrollment, even when we aren't enrolled.
+// The previous state of a feature before enrollment.
+// ⚠️ Attention : Changes to this type should be accompanied by a new test  ⚠️
+// ⚠️ in `src/stateful/tests/test_enrollment_bw_compat.rs` below, and may require a DB migration. ⚠️
```

Comment:
This comment should also be on PreviousState

Reply:
Thanks!